proto/motor/v1/request.proto

```diff
@@ -168,3 +170,19 @@ message SeachBucketContentBySchemaRequest {
 }
 
 message StatRequest {}
+
+message UploadDocumentRequest {
```

Comment:
Why is this in this PR?

Reply:
think it was from a merger i did and committed by mistake, good catch. removing this